docs/Dialects/FIRRTL/FIRRTLAnnotations.md

```diff
@@ -299,6 +299,31 @@ Example:
 }
 ```
 
+### Convention
+
+| Property   | Type   | Description                 |
+| ---------- | ------ | --------------------------- |
+| class      | string | `firrtl.module.Convention`  |
+| convention | string | `scalarize` or `internal`   |
+| target     | string | Reference target            |
+
+Specify the instantiation convention for a module. The instantiation convention
+controls how a module's ports are transformed, and how that module can be
+instantiated, in the output format.
+
+The options are:
+- `internal`: Use the unstable internal linkage convention.
+- `scalarized`: Convert aggregate ports (i.e. vector or bundles) into multiple
+  ground-typed ports.
+
+```json
+{
+  "class": "firrtl.linkage.Convention",
```

Comment:
This doesn't match the documentation above.  However it's called, they should match.  Sorry for the noise on re:feedback.

Reply:
Thanks yes, totally forgot to change these. I promise to eventually eradicate the word linkage from this PR. I still need a real class name that we can use in chisel,  suggestions anyone?